src/domains/core/app.container.js

```diff
@@ -19,6 +21,8 @@ export const App = () => {
     } else {
       dispatch(recoverSession());
     }
+
+    dispatch(fetchFavoritesSongs());
```

Comment:
Dispatching this action here will cause `401` error (and an endless loop of redirects) if the user tries to access the `/login` route with not auth.